search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,301 @@
+* The CMIS part of the query and `CONTAINS()` part are melded together into a single abstract query representation. By default, in CMIS the `CONTAINS()` expression implies full text search, so the queries will go to the Elasticsearch index.
+* The database does not score. It will return results in some order that depends on the query plan, unless you ask for specific ordering. A three part `OR` query, where some documents match more than one constraint, is treated as equal. For Elasticsearch index queries, the more parts of an `OR` match, the higher is the score. The docs that match more optional parts of the query will come higher up.
+* Queries from Share will not use TMDQ as they will most likely have a full text part to the query and ask for facets.
+* Exact term search will behave differently when executed against the database or the search index. This is due to how tokenisation is applied to strings in the search index, for more see [Exact Term Queries](https://hub.alfresco.com/t5/alfresco-content-services-blog/exact-term-queries-in-search-services-2-0/ba-p/302200).
+
+### Exact match and patterns
+
+TMDQ can support exact match on all properties (subject to database collation) regardless of the property index configuration in the data model. All text properties can support pattern matching. The database index supports a fixed number of leading characters. The database store a maximum string size before it overflows to another form. Only short form strings can be used in database queries.
+
+Elasticsearch supports exact match on all non-text properties. Text properties only support exact and pattern matches if set to tokenised `both` or `false` in the data model. Elasticsearch provides supports values up to approximately 32,700 UTF-8 bytes.
+
+The following specific CMIS QL fields are supported:
+
+* `cmis:parentId`
+* `cmis:objectId`
+* `cmis:objectTypeId`
+* `cmis:baseTypeId`
+* `cmis:contentStreamMimeType`
+* `cmis:contentStreamLength`
+
+The following CMIS QL comparison operators are supported:
+
+* `=`, `!=,` `<>`, `<`, `<=`, `>`, `>=`
+* `IN`, `NOT IN`, `LIKE`
+
+The following AFTS exact matches and patterns are supported:
+
+* `=<field>:term`
+* `=<field>:ter*`
+* `=<field>:*erm`
+
+### Support for special fields in TMDQ using AFTS
+
+* PARENT
+* TYPE
+* ASPECT
+* EXACTTYPE
+* EXACTASPECT
+
+> **Note:** CMIS QL does not support any use of CONTAINS() using the database.
```

Comment:
Change CONTAINS() to `CONTAINS()`, I think.

Reply:
Changed cheers